akka-persistence-query/src/main/scala/akka/persistence/query/typed/javadsl/EventsBySliceFirehoseReadJournal.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.query.typed.javadsl
+
+import java.time.Instant
+import java.util
+import java.util.Optional
+import java.util.concurrent.CompletionStage
+
+import scala.compat.java8.FutureConverters._
+import scala.compat.java8.OptionConverters._
+
+import akka.NotUsed
+import akka.dispatch.ExecutionContexts
+import akka.japi.Pair
+import akka.persistence.query.Offset
+import akka.persistence.query.javadsl.ReadJournal
+import akka.persistence.query.typed.EventEnvelope
+import akka.persistence.query.typed.scaladsl
+import akka.stream.javadsl.Source
+
+object EventsBySliceFirehoseReadJournal {
+  val Identifier: String = scaladsl.EventsBySliceFirehoseReadJournal.Identifier
+}
+
+class EventsBySliceFirehoseReadJournal(delegate: scaladsl.EventsBySliceFirehoseReadJournal)
```

Comment:
```suggestion
final class EventsBySliceFirehoseReadJournal(delegate: scaladsl.EventsBySliceFirehoseReadJournal)
```

I thought we have several "Not for user construction, lookup using xxx" but couldn't find any now that I was looking, only the not for user extension. Maybe a good idea anyway.